### ISSUE TITLE
HTML report doesn't generates when gradle-lombok plugins is also applied

### ISSUE THREAD

**Issue by daggerok** [spotbugs/spotbugs-gradle-plugin#94]
Hello, this issue comes from that one: #32 

**what's wrong**

when gradle lombok plugin is applied, spotbugs-gradle-plugin generates XML reports instead of configured HTML format

**how to reproduce**

to reproduce a bug, please do next:

1) clone [daggerok/spotbugs-gradle-plugin-multi-project-example](https://github.com/daggerok/spotbugs-gradle-plugin-multi-project-example):
```bash
git clone https://github.com/daggerok/spotbugs-gradle-plugin-multi-project-example
```

2) uncomment [line 25](https://github.com/daggerok/spotbugs-gradle-plugin-multi-project-example/blob/7748c6604a0bd54a7ab0152709f4e681f943d862/build.gradle#L25) in root `build.gradle` file:
```gradle
//  id 'io.franzbecker.gradle-lombok' version '1.14' apply false
```

3) run spotbugs analysis:
```bash
./gradlew check -PignoreBugs
```

4) and finally, verify wrong reports format:
```bash
tree ./build/spotbugs
./build/spotbugs
├── more-sub-projects
│   ├── main.xml
│   └── test.xml
├── more-sub-projects-one-more-sub-project
│   ├── main.xml
│   └── test.xml
└── sub-project
    ├── main.xml
    └── test.xml
```

as you can see, XML reports was generated instead of configured html format. As soon [line 25]( https://github.com/daggerok/spotbugs-gradle-plugin-multi-project-example/blob/7748c6604a0bd54a7ab0152709f4e681f943d862/build.gradle#L25) will be commented out, html reports will be generated as expected.

---
Regards,
Maksim

**Comment by simPod** [comment]
I have also discovered that it generates HTML reports but only up to version 1.6.8.

When upgraded to 1.6.9+ it generates XML 🤔

**Comment by daggerok** [comment]
@simPod checkout my solution from [1.6.9 branch](https://github.com/daggerok/spotbugs-gradle-plugin-multi-project-example/tree/1.6.9). It's generates html report with 1.6.9 version without lombok plugin applied, but it's not [when lombok plugin applied (see branch 1.6.9-lombok)](https://github.com/daggerok/spotbugs-gradle-plugin-multi-project-example/tree/1.6.9-lombok)

**Comment by simPod** [comment]
@daggerok I see. It was probably misbehaving for me and I had to restart gradle daemons. Now it works with 1.7.1 as well.

I have similar issue as you. Having this makes spotbugs generate xml no matter what.

```gradle
buildscript {
    dependencies {
        classpath 'com.github.jengelman.gradle.plugins:shadow:4.0.3'
    }
}
```

Removing `classpath 'com.github.jengelman.gradle.plugins:shadow:4.0.3'` results in html report

**Comment by simPod** [comment]
Also any other plugin, like spring boot causes the same

**Comment by KengoTODA** [comment]
I cannot reproduce this problem with latest versions:

* Gradle 5.6 (to use spotbugs-gradle-plugin v4.x)
* spotbugs 4.0.6
* spotbugs-gradle-plugin 4.4.3

please feel free to reopen this issue when you could reproduce with them.
